bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+be improved to progress toward readiness. Pull requests that are proposing off-topic or unserious ideas or have
+stopped to make progress may be closed.
+
+When the proposal has been assigned a number and is complete, a BIP editor will merge it to the BIPs repository.
+After the BIP is merged to the repository, its main focus should no longer shift significantly, even while the author
+may continue to update the proposal as necessary in the git repository. Updates to merged documents by the author should also be
+submitted as pull requests.
+
+#### Proposed[^proposed]
+
+When the author is confident that their BIP represents a net improvement, is clear, comprehensive, has a working
+reference implementation (if applicable), and is ready for adoption by the bitcoin community, they may update the BIP’s
+status to Proposed to indicate that they recommend adoption or implementation of the BIP. Where applicable, the
+author must ensure that any proposed specification is solid, not unduly complicated, and definitive. Subsequently, the
+BIP’s content should only be adjusted in minor details, e.g. to improve language, clarify ambiguities, backfill
+omissions in the specification, add test vectors for edge cases, or address other issues discovered as the BIP is
+adopted.
+
+A Proposed BIP can only move to Active or Abandoned. Any necessary changes to the specification should be
+minimal and interfere as little as possible with ongoing adoption. If a Proposed BIP is found to need
+substantial functional changes, it may be preferable to move it to Abandoned[^new-BIP], and to start a new BIP with the changes
+instead. Otherwise, it could cause confusion as to what being compliant with the BIP means.
+
+#### Active
+
+A settled[^settled] BIP may be advanced to Active upon request by
+any community member with evidence that the idea described in the BIP is in active use. Such evidence includes
+for example: two or more projects having deployed support for the BIP in mainnet software releases, a soft fork
+proposal’s activation criteria having been met on the network, or rough consensus for the BIP having been demonstrated.
+
+At that point, the BIP should be considered final and any breaking changes to the BIP should be proposed as a new
+separate BIP.[^new-BIP]
+
+##### Process BIPs
+
+A Process BIP may change status from Proposed to Active when it achieves rough consensus on the mailing list. Such a
+proposal is said to have rough consensus if it has been open to discussion on the development mailing list for at least
+one month, and no person maintains any unaddressed substantiated objections to it. Addressed or obstructive objections
+may be ignored/overruled by general agreement that they have been sufficiently addressed, but clear reasoning must be
+given in such circumstances. Active Process BIPs may be modified indefinitely as long as a proposed modification
+has rough consensus per the same criteria.[^living-documents]
+
+##### Revisions
```

Comment:
I think this should be bumped up to "###" and moved to just before the "Adoption of Proposals" heading. Possibly also name it as "Changelog" instead of "Revisions" ?

Reply:
Sure, sounds good. I moved it just before "Adoption of Proposals" bumped it to "###" and renamed it to "Changelog".